### ISSUE TITLE
✔️ 🐞 Bug Report: Forms below Loop Grid can not be sent/submitted - [ED-9839]

### ISSUE THREAD

**Issue by Flickimp** [elementor/elementor#21432]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

See this link; [https://staging14.websquadron.co.uk/image/](https://staging14.websquadron.co.uk/image/

The form above and below the Loop Grid are 'exactly' the same. 

But the form below the Loop Grid will not send. When the Loop Grid is removed then it's fine.

I have tested this on several websites with no other optimisation or snippets added.

**The issue is any version since 3.11
If you roll back to Pre 3.11 then it is fine.**

### Steps to reproduce

1. Add a basic Elementor Form
2. Add a Loop Grid with a template for a basic post.
3. Add another basic Elementor Form (or copy from Step 1)

Form 1 will send. But the second Form will not send.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.39-42
	PHP Version: 7.4.33
	PHP Memory Limit: 768M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://staging14.websquadron.co.uk
	Home URL: https://staging14.websquadron.co.uk
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 256M
	Max Memory limit: 768M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Admin Email: BATSLincs@outlook.com
	Debug Mode: Inactive

== Theme ==
	Name: Bricks
	Version: 1.6.2
	Author: Bricks
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36 Edg/110.0.1587.50

== Active Plugins ==
	Elementor
		Version: 3.11.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.11.2
		Author: Elementor.com


== Elements Usage ==
	
	wp-page : 3
		container : 3
		form : 2
		loop-grid : 1


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Inactive
	Improved Asset Loading: Inactive
	Improved CSS Loading: Inactive
	Inline Font Icons: Inactive
	Accessibility Improvements: Inactive
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Inactive
	Editor V2: Inactive by default
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive
	Save as Default: Inactive
	Page Transitions: Inactive
	Notes: Inactive
	Loop: Active
	Form Submissions: Inactive
	Scroll Snap: Inactive
	Mega Menu: Inactive by default


== Log ==
	
Log: showing 20 of 262023-01-24 19:10:26 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.1',
  'to' => '3.11.0-beta1',
)]
2023-01-24 19:10:26 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.1',
  'to' => '3.11.0-beta1',
)]
2023-01-24 20:46:12 [info] elementor-pro::elementor_pro_updater Started 
2023-01-24 20:46:12 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-01-24 20:46:12 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-01-24 20:46:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.2',
  'to' => '3.11.0-beta1',
)]
2023-01-24 20:46:12 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.2',
  'to' => '3.11.0-beta1',
)]
2023-02-08 20:08:24 [info] elementor-pro::elementor_pro_updater Started 
2023-02-08 20:08:24 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-08 20:08:24 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-08 20:08:24 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.0-beta1',
  'to' => '3.11.0-beta3',
)]
2023-02-08 20:08:24 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.0-beta1',
  'to' => '3.11.0-beta3',
)]
2023-02-22 21:47:07 [info] elementor::elementor_updater Started 
2023-02-22 21:47:07 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-22 21:47:07 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.0-beta1',
  'to' => '3.11.2',
)]
2023-02-22 21:47:12 [info] elementor-pro::elementor_pro_updater Started 
2023-02-22 21:47:12 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-22 21:47:12 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-22 21:47:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.0-beta3',
  'to' => '3.11.2',
)]
2023-02-22 21:47:12 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.0-beta3',
  'to' => '3.11.2',
)]

JS: showing 13 of 13JS: 2023-01-18 10:07:09 [error X 1][https://staging14.websquadron.co.uk/wp-content/plugins/elementor/assets/js/new-template.min.js?ver=3.10.1:2:2932] elementor_new_template_form_controls is not defined 
JS: 2023-01-23 09:45:03 [error X 9][https://staging14.websquadron.co.uk/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-01-24 20:50:54 [error X 3][https://staging14.websquadron.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.0-beta1:3:1086973] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-02-04 09:29:47 [error X 1][https://staging14.websquadron.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.0-beta1:3:670621] this.model.isValidChild is not a function 
JS: 2023-02-05 11:10:56 [error X 8][https://staging14.websquadron.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.0-beta1:3:791350] Cannot read properties of undefined (reading 'localized') 
JS: 2023-02-05 13:47:00 [error X 41][https://staging14.websquadron.co.uk/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property 'guid' on string 'c2517widgetsticky' 
JS: 2023-02-05 14:03:18 [error X 1][https://staging14.websquadron.co.uk/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property 'guid' on string 'c7720containersticky' 
JS: 2023-02-05 14:03:18 [error X 3][https://staging14.websquadron.co.uk/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property 'guid' on string 'c8378widgetsticky' 
JS: 2023-02-05 14:03:34 [error X 3][https://staging14.websquadron.co.uk/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property 'guid' on string 'c9966widgetsticky' 
JS: 2023-02-05 14:04:23 [error X 14][https://staging14.websquadron.co.uk/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property 'guid' on string 'c13248widgetsticky' 
JS: 2023-02-05 14:05:15 [error X 2][https://staging14.websquadron.co.uk/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property 'guid' on string 'c14333widgetsticky' 
JS: 2023-02-05 14:07:09 [error X 28][https://staging14.websquadron.co.uk/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property 'guid' on string 'c21496widgetsticky' 
JS: 2023-02-05 14:13:29 [error X 3][https://staging14.websquadron.co.uk/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property 'guid' on string 'c26962widgetsticky' 

PHP: showing 4 of 4PHP: 2023-01-18 19:25:36 [notice X 17][/home/customer/www/staging14.websquadron.co.uk/public_html/wp-content/plugins/elementor/app/modules/import-export/module.php::628] Trying to get property 'label' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-22 17:39:52 [notice X 5][/home/customer/www/staging14.websquadron.co.uk/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::172] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-25 21:29:04 [notice X 4][/home/customer/www/staging14.websquadron.co.uk/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::168] Undefined index: editor_post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-22 21:47:08 [error X 1][/home/customer/www/staging14.websquadron.co.uk/public_html/wp-content/plugins/elementor-pro/modules/popup/module.php::102] Uncaught Error: Class 'ElementorPro\Modules\Popup\Tag' not found in /home/customer/www/staging14.websquadron.co.uk/public_html/wp-content/plugins/elementor-pro/modules/popup/module.php:102
Stack trace:
#0 /home/customer/www/staging14.websquadron.co.uk/public_html/wp-includes/class-wp-hook.php(308): ElementorPro\Modules\Popup\Module->register_tag(Object(Elementor\Core\DynamicTags\Manager))
#1 /home/customer/www/staging14.websquadron.co.uk/public_html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters(NULL, Array)
#2 /home/customer/www/staging14.websquadron.co.uk/public_html/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#3 /home/customer/www/staging14.websquadron.co.uk/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(271): do_action('elementor/dynam...', Object(Elementor\Core\DynamicTags\Manager))
#4 /home/customer/www/staging14.websquadron.co.uk/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(228): Elementor\Core\DynamicTags\Manager->get_tags()
#5 / [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

				
````
</details>

**Comment by hamzakarcic** [comment]
Same here

**Comment by nicholaszein** [comment]
Hi @Flickimp! 

Thank you for submitting your issue! 🙏 

We can confirm this issue has been replicated and the team is already working on fixing this issue.

I will update this issue when we have more information. Thank you for the detailed information and for the patience! 

Best regards

**Comment by Flickimp** [comment]
Muchos gracias :)

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.11.3**! 🥳 

Thanks @Flickimp for reporting this issue!

✔️ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
